### PR TITLE
Remove pre-push hook

### PR DESCRIPTION
**Pull request by swissspidy**
Avoids running unnecessary tests on push. We have CI for that

### CODE REVIEW

**Comment by swissspidy** [comment]
Ah the comma 🤦‍♂️

**Comment by spacedmonkey** [comment]
@swissspidy Build still failing.

**Comment by swissspidy** [comment]
Because I haven‘t fixed the errant comma yet :-)

File: package.json [hunk 1]
```diff
@@ -121,7 +121,6 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
```
Comment (spacedmonkey) [inline package.json]:
```suggestion
      "pre-commit": "lint-staged"
```

Comment (swissspidy) [reply package.json]: Thanks!

**Review by miina — APPROVED** [review]

**Pull request merged**